config/vufind/Folio.ini

```diff
@@ -181,6 +181,12 @@ in_transit[] = "Open - Awaiting delivery"
 ; retrieved from FOLIO will be retained.
 ;vufind_sort = "enumchron"
 
+; If set to true (default) and there are no items attached to a FOLIO holdings record, 
```

Comment:
Would it be safer to default to false to prevent unexpected changes in behavior? Or do you think the benefit is great enough that we should change the default behavior? (At Villanova, I'm pretty sure we _don't_ want this, so I'm possibly biased ;-) ).

Reply:
6m ago I would have locally enabled this, now after some data changes I would locally disable.  The default is a hard call, this is very useful behavior when you have a record structured this way, but it's also not necessary intuitive, and something that a library should maybe consider when setting up FOLIO integration based on their own holdings.  So I guess I would default to false but I don't feel strongly about it.

Reply:
Maybe the way to go is false for now (if we want this in release 10.1) for a more backward-compatible approach, and we can revisit the default in the future if demand seems to push us that way.

My main concern if we change the default is that we'll get an influx of support requests about "why are these phantom items showing up?" Since this feature generates "fake" data (at least according to VuFind's model of things), it makes more sense to me as an opt-in, where people turn it on after they understand what it is for, rather than having to figure out the behavior when they unexpectedly encounter it.

But I'm still open to counter-arguments if anyone feels strongly the other way.

Reply:
Good enough reasoning for me. New merge coming.